plugins/TodayAtMun/__init__.py

```diff
@@ -15,6 +14,15 @@ def __init__(self, manifest, bot: commands.Bot):
         self.parse = DiaryParser()
         self.today_fun = Today(self.parse.diary)
 
+    async def today_embed_template(self):
```

Comment:
This function does not await anything, so it is not necessary for it to be an async function